packages/mdc-textfield/README.md

```diff
@@ -207,12 +199,12 @@ apply these mixins with CSS selectors such as `.foo-text-field.mdc-text-field--i
 
 > _NOTE_: the `mdc-line-ripple-color` mixin should be applied from the not focused class `foo-text-field:not(.mdc-text-field--focused)`).
 
-#### Mixins for Text Field Box
+#### Mixins for Text Field
 
 Mixin | Description
 --- | ---
-`mdc-text-field-box-corner-radius($radius)` | Customizes the border radius for the text field box variant.
-`mdc-text-field-box-fill-color($color)` | Customizes the background color of the text field box.
+`mdc-text-field-corner-radius($radius)` | Customizes the border radius for the text field.
+`mdc-text-field-fill-color($color)` | Customizes the background color of the text field.
 
```

Comment:
Consolidate the "Other Mixins" table into the "Mixins for Text Field" table. They don't need to be separate anymore since there's no more Box variant.

Reply:
Complete!